bufbuild/connect-crosstest
# Add grpc c++ client conformance tests

The minimal footprint to get a grpc-based binary that can run the conformance tests. 
To test run:
 - run a test server, for example `go run cmd/serverconnect/main.go --insecure --h1port=8080 --h2port=8081`
 - run the conformance tests `PORT=8081 bazel run //cc:grpc_client_test`

Will require follow up PRs to fully integrate into crosstest docker images and CI.

Also note that the grpc client does not properly convert 404's into UNIMPLEMENTED, instead returning UNKNOWN.
